api-reference/docstubs/api/user-delta.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "delta"
+description: ""
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# delta
+
+Namespace: microsoft.graph
+
+
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/concepts/permissions-reference.md).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|**TODO: Determine scopes **|
+|Delegated (personal Microsoft account)|Not supported.|
+|Application|**TODO: Determine AppOnly scopes **|
+
+## HTTP request
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+GET /users/delta
+GET /me/managedDevices/{managedDeviceId}/users/delta
+```
+
+## Request headers
+|Name|Description|
+|:---|:---|
+|Authorization|Bearer {token}.Required|
+
+## Request body
+Do not supply a request body for this method.
+
+## Response
+If successful, this function returns a `200 OK` response code and a [user](../resources/user.md) collection in the response body.
```

Comment:
I suppose for the delta topic to be educational, the response section will need to explain in more detail what would be returned. Not sure if that volume of explanation would go in as annotation, even though technically, yes, it's just difficult to author in an XML file **out of context** with the rest of the method topic.

GLOBAL: 
That, btw, is what I am most uncertain of about relying on partners to enter documentation in the CSDL file with the purpose of fully generating reference docs. Authoring outside the context of the actual method topic as in a **WYSIWYG  authoring environment**, and doing specific section in an XML file, makes it easy to overlook the full picture of the topic. 

Back to this delta method topic:
For the response, there is the default case which is whatever specified in the initial delta request, and there's the case of returning only the changed properties, when the Prefer header is specified.
So the Response section needs manual

Reply:
Agreed - this is likely always a manual, or part manual topic.  Agree on the authoring aspects as well.  We'll need to work this out.